package.json

```diff
@@ -30,7 +30,8 @@
     "immutable": "3.8.2",
     "react-dropzone": "10.1.10",
     "react-flexbox-grid": "2.1.2",
-    "react-popover": "^0.5.10"
+    "react-popover": "^0.5.10",
+    "recharts": "^2.0.0-beta.1"
```

Comment:
Can we use recharts as peerDependencies to keep it in sync with web-ui-common?

"peerDependencies": { "recharts": ">=1.8.5" }
"devDependencies": { "recharts": "1.8.5" }